skbio/io/format/tests/test_stockholm.py

```diff
@@ -376,5 +389,109 @@ def test_handles_missing_metadata_efficiently(self):
         self.assertIsNone(msa[0]._positional_metadata)
 
 
+class TestStockholmWriter(unittest.TestCase):
+    def test_msa_to_stockholm_extensive(self):
+        fp = get_data_path('stockholm_all_data_types')
+        msa = TabularMSA([DNA('GAGGCCATGCCCAGGTGAAG',
+                              metadata={'DT': 'February 1, 2016'}),
```

Comment:
Can you update this test case to include writing more than one GS, GR, and GC feature? That way any problems with looping over metadata will be caught.